mrchecker-framework-modules/mrchecker-webapi-module/src/test/java/com/capgemini/mrchecker/webapi/httpbin/Base64Test.java

```diff
@@ -0,0 +1,44 @@
+package com.capgemini.mrchecker.webapi.httpbin;
+
+import com.capgemini.mrchecker.test.core.logger.BFLogger;
+import com.capgemini.mrchecker.webapi.BasePageWebApiTest;
+import com.capgemini.mrchecker.webapi.pages.httbin.Base64Page;
+import io.restassured.response.Response;
+import org.junit.Test;
+
+import java.text.MessageFormat;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class Base64Test extends BasePageWebApiTest {
+	private Base64Page base64Page = new Base64Page();
+
+	@Test
+	public void base64DecodeAwesome() {
+		String validBase64Data = "SFRUUEJJTiBpcyBhd2Vzb21l";
+		String validBase64Decode = "HTTPBIN is awesome";
+		BFLogger.logInfo(MessageFormat.format("Step 1 - Sending GET query to {0} with valid base 64 data: {1}", base64Page.getEndpoint(), validBase64Data));
+		Response response = base64Page.sendGETQuery(validBase64Data);
```

Comment:
Please take a look on duplicated blocks of code and then wrap them as a separate private method to reuse.